db/data/development-data/index.js

```diff
@@ -1,4 +1,17 @@
-exports.categoryData = require('./categories.js');
-exports.commentData = require('./comments.js');
-exports.reviewData = require('./reviews.js');
-exports.userData = require('./users.js');
+exports.categoryData = require("./categories.js");
+exports.commentData = require("./comments.js");
+exports.reviewData = require("./reviews.js");
+exports.userData = require("./users.js");
+
+const { Pool } = require("pg");
+const ENV = process.env.NODE_ENV || "development";
+
+const pathToEnvFile = `${__dirname}/.env.${ENV}`;
+
+console.log(pathToEnvFile);
+
+require("dotenv").config({
+  path: pathToEnvFile,
+});
+
+module.exports = new Pool();
```

Comment:
This connection logic is already setup for you in the `connection.js` file - no need to add it here! In fact, trying to export a database connection will mess with the behaviour of the rest of your app. Just remove it 🙂